transit_odp/publish/views/timetable/seasonal_services.py

```diff
@@ -177,12 +178,18 @@ def get_form_kwargs(self):
         return kwargs
 
     def get_success_url(self):
+        org_id = self.kwargs["pk1"]
         page = self.request.GET.get("page", None)
+        seasonal_services = SeasonalService.objects.filter(
```

Comment:
This is already defined in `get_queryset`. I wonder if this means `self.queryset` is now defined or `self.object_list`

Reply:
I am in DeleteView there are no `get_queryset`, `self.queryset` and `self.object_list`